mesheryctl/pkg/utils/helpers.go

```diff
@@ -31,10 +31,12 @@ const (
 	dockerComposeBinary         = "/usr/local/bin/docker-compose"
 
 	// Usage URLs
-	rootUsageURL   = "https://meshery.layer5.io/docs/guides/mesheryctl#global-commands-and-flags"
-	perfUsageURL   = "https://meshery.layer5.io/docs/guides/mesheryctl#performance-management"
-	systemUsageURL = "https://meshery.layer5.io/docs/guides/mesheryctl#meshery-lifecycle-management"
-	meshUsageURL   = "https://meshery.layer5.io/docs/guides/mesheryctl#service-mesh-lifecycle-management"
+	DOCS_BASEURL = "https://meshery.layer5.io/docs/guides/mesheryctl"
```

Comment:
Only put docs base URL in constants & please follow camel case

Reply:
An example being ```docsBaseURL = "https://meshery.layer5.io/docs/"```

Right, @kushthedude?